samples/AzureFunctionsApp/Entities/Lifetime.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Net;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Client.Entities;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Logging;
+
+namespace AzureFunctionsApp.Entities;
+
+/// <summary>
+/// Example showing the lifetime of an entity. An entity is initialized on the first operation it receives and then
+/// is considered deleted when <see cref="TaskEntity{TState}.State"/> is <c>null</c> at the end of an operation.
+/// </summary>
```

Comment:
Possibly mention:
It is also possible to design an entity that remains stateless, by overriding InitializeState to return null, and never assigning a non-null state.